newIDE/app/src/InstancesEditor/index.js

```diff
@@ -132,6 +135,9 @@ export default class InstancesEditor extends Component<Props> {
   nextFrame: AnimationFrameID;
   contextMenuLongTouchTimeoutID: TimeoutID;
   hasCursorMovedSinceItIsDown = false;
+  duplicateInstanceLastCursorRetrieved = false;
```

Comment:
I don't understand what this variable is supposed to do with the naming. Can you name it like a question, just like `hasCursorMovedSinceItIsDown`?

Reply:
Furthermore, I don't think you need this variable. What you could do is defining  duplicateInstanceLastCursorX/Y as a number or null variable.

```js
duplicateInstanceLastCursorX: number | null = null;
```

And, on the first call of `onMoveInstances`, you get the last cursor position and store it in those variables if they are `null`.
And when the duplication is done, you set them back to `null`.